cornellh4i/ithaca-recovery
# Atoms/date time picker fix

### Description 
Implemented fixes to TimePicker such as displaying the time on a 12-hour-clock format, setting default times, making the time period stay the same when the starting time changes, and making sure the ending time cannot be set as before the starting time.

### Testing 
1. Navigate to http://localhost:3000/test
2. Scroll to the bottom of the page, and observe the DatePicker/TimePicker section
3. Try to input a time of 14:30 and observe how it will automatically convert 2:30 pm
4. Input any starting time and the ending time will automatically be set to an hour after
5. Try to change the ending time to something earlier than the starting time, it won't let you input it
6. Change the ending time to something valid, then change the starting time, and the ending time will automatically change to retain the same time period

### Notes
related to issue #60
